utils/build-script-impl

```diff
@@ -740,20 +743,26 @@ CROSS_TOOLS_DEPLOYMENT_TARGETS=()
 
 # Determine the native deployment target for the build machine, that will be
 # used to jumpstart the standard library build when cross-compiling.
-case "$(uname -s)" in
-    Linux)
+# FIXME: This will not work while trying to cross-compile
+# It gets the architecture from the system processor arch.
```

Comment:
You can choose a head branch when opening the pull request. The changes on this head branch (in your case master) will be tracked in the pull request. 
Normally, I create a separate branch for each pull-request *before* opening it.

So, you'd either have to continue working on your master or close and resubmit the pull request with your `armv7` branch selected as the HEAD. I don't think it's possible to change the head branch once a pull request has been created.